src/Microsoft.Azure.EventHubs.Processor/Microsoft.Azure.EventHubs.Processor.csproj

```diff
@@ -29,7 +29,6 @@
   </PropertyGroup>
 
   <PropertyGroup Condition="'$(TargetFramework)' == 'uap10.0'">
-    <DefineConstants>$(DefineConstants);UAP10_0</DefineConstants>
```

Comment:
Let's remove this change.

Reply:
EPH code doesn't use this constant, so safe to remove.

---
In reply to: [156541541](https://github.com/Azure/azure-event-hubs-dotnet/pull/225#discussion_r156541541) [](ancestors = 156541541)